code/drasil-website/lib/Drasil/Website/About.hs

```diff
@@ -99,6 +100,10 @@ aboutParagraph8 paperGOOL = S "As described in the" +:+ namedRef paperGOOL (S "G
   \generation, allowing Drasil to more efficiently generate code in several languages, including Python, Java, C-Sharp, and C++."
 
 -- | Paragraph providing a link to Drasil papers and documents
-aboutParagraph9 :: Reference -> Sentence
-aboutParagraph9 papersWiki = S "A list of papers and documents written about Drasil can be found in the" +:+ namedRef papersWiki (S "Drasil Papers \
-  \and Documents") +:+ S "wiki page"
\ No newline at end of file
+aboutParagraph9 :: Reference -> Reference -> Reference -> Reference -> Sentence
+aboutParagraph9 papersWiki icsePositionPaper danPoster wellUnderstoodPaper = 
+  S "A list of papers and documents written about Drasil can be found in the" +:+ 
+  namedRef papersWiki (S "Drasil Papers and Documents") +:+ S "wiki page. In particular, there is a" +:+ 
+  namedRef icsePositionPaper (S "Old Position Paper") +:+ S "outlining our original ideas, a" +:+ 
+  namedRef danPoster (S "Drasil Poster") +:+ S ", and a" +:+ 
+  namedRef wellUnderstoodPaper (S "Well-Understood Paper") +:+ S "discussing key concepts."
```

Comment:
This should use `foldlList`, or something similar. I seem to remember describing how to do this, as well as some other improvements, in a previous PR; you can use that as reference!